tests/suites/require.js

```diff
@@ -15,11 +15,15 @@ casper.test.begin('Javascript module loading', 1, function(test) {
 
 casper.test.begin('CoffeeScript module loading', 1, function(test) {
     var csmod;
-    try {
-        csmod = require(fs.pathJoin(modroot, 'csmodule'));
-        test.assertTrue(csmod.ok, 'require() patched version can load a coffeescript module');
-    } catch (e) {
-        test.fail('require() patched version can load a coffeescript module');
+    if (Object.keys(require.extensions).indexOf('.coffee') !== -1) {
```

Comment:
Also, as there's a single specific test here it's probably worth moving that if statement around the `casper.test` call itself, as it's been done elsewhere already. Adding a comment to explain why we're excluding coffee files might be worthwhile too.